.gitignore

```diff
@@ -2,4 +2,6 @@
 *.exe
 .env
 .vscode
-logs
\ No newline at end of file
+logs
+.mongo_pw
```

Comment:
Was adding back these two files in the gitignore a mistake? .mongo_pw and .mongo_user not used anymore. Other than that looks good

Reply:
Done, I have removed these two.